Formula/jenkins.rb

```diff
@@ -56,7 +56,7 @@ def plist; <<-EOS.undent
 
   test do
     ENV["JENKINS_HOME"] = testpath
-    ENV["_JAVA_OPTIONS"] = "-Djava.io.tmpdir=#{testpath}"
+    ENV.prepend "_JAVA_OPTIONS", "-Djava.io.tmpdir=#{testpath}"
```

Comment:
Technically this change is pedantry I think.

Reply:
In what sense? Maybe I'm misunderstanding what you're saying here so I'll ask rather than guess :smile:

Reply:
We've never previously needed to set `-Duser.home` for this test so worrying about `=` vs. `prepend` should be a distinction without a difference.

Reply:
Oh, yeah. This one is literally just a case of not wanting to mix & match how we handle `_JAVA_OPTIONS` because it gives people not always correct examples to lean on in future. I can flip it back though if you prefer.